examples/RL/create_gym_example.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import gym
+import l5kit.environment
+
+
+# TODO: Create a Google colab notebook for environment creation and training.
+
+# By setting the L5KIT_DATA_FOLDER variable, we can point the script
+# to the folder where the data lies.
+os.environ["L5KIT_DATA_FOLDER"] = os.environ["HOME"] + '/level5_data'
+env_config_path = "./config.yaml"
+
+# call
+env = gym.make('L5-CLE-v0', env_config_path=env_config_path)
```

Comment:
The default `env_config_path` is `None` so that environment can be registered (as per our discussion today).

I personally prefer to explicitly point out the config file being used for this example.

Reply:
That's fine, thanks !